templates/components/styles/notes.html.twig

```diff
@@ -1,7 +1,7 @@
 {%- macro notes(type, title, notes) -%}
     {% if (notes|length > 0) %}
         <div class="l-box note note--{{ type }}" role="{{ type == 'error' ? 'alert' : 'status' }}" aria-labelledby="{{ type }}-summary-title" tabindex="-1"
-             data-component="{{ type }}-summary">
+             data-component="{{ type }}-summary" data-anchor="no">
```

Comment:
@jean-gui This should not be needed here, because:

- the script only targets headings within `<main>`
- the parent container for this component is `<aside>` and in the templates I do not believe it is nested within `<main>`. It is not intended to be, as per the original designs.